ucm2/Qualcomm/x1e80100/HiFi.conf

```diff
@@ -53,3 +53,17 @@ SectionDevice."Headphones" {
 		JackHWMute "Speaker"
 	}
 }
+
+SectionDevice."DMic01" {
```

Comment:
Naming does not follow convention: https://github.com/alsa-project/alsa-lib/blob/master/include/use-case.h#L135 . Description should be put to the comment only.

Reply:
Ack, I'll fix it.